packages/matchbox/src/components/Pager/Pager.js

```diff
@@ -13,9 +14,14 @@ const StyledPager = styled('div')`
 `;
 
 function Pager(props) {
-  const { children, ...rest } = props;
+  const { children, id, 'data-id': dataId, ...rest } = props;
```

Comment:
Do we want to allow className here or ok without it?

Reply:
I looked at the UI and looks like we dont need to support it. We should avoid supporting restyling components anyway